src/ui/libs/schematic-sdk/index.ts

```diff
@@ -39,6 +40,14 @@ export type DatalensSdk<TSchema extends SchemasByScope> = (<T>(
 
 const sdkConfig: SdkConfig = {
     csrfToken: Utils.getCSRFToken() || '',
+    axiosConfig: {
```

Comment:
What would delay be equal to for this retries?

Reply:
discussed offline, added